drivers/amazonec2/stub_test.go

```diff
@@ -96,27 +96,30 @@ type fakeEC2SecurityGroupTestRecorder struct {
 	mock.Mock
 }
 
-func (f *fakeEC2SecurityGroupTestRecorder) DescribeSecurityGroups(
-	input *ec2.DescribeSecurityGroupsInput) (*ec2.DescribeSecurityGroupsOutput, error) {
-
+func (f *fakeEC2SecurityGroupTestRecorder) DescribeSecurityGroups(input *ec2.DescribeSecurityGroupsInput) (*ec2.DescribeSecurityGroupsOutput, error) {
 	result := f.Called(input)
-	value, _ := result.Get(0).(*ec2.DescribeSecurityGroupsOutput)
+	value, ok := result.Get(0).(*ec2.DescribeSecurityGroupsOutput)
```

Comment:
Probably best being ignored for now then.  It's my instinct to never swallow `err` or `ok` but since it's for a test I think it's probably OK.